src/VkBootstrap.h

```diff
@@ -318,6 +318,8 @@ struct Instance {
     VkAllocationCallbacks* allocation_callbacks = nullptr;
     PFN_vkGetInstanceProcAddr fp_vkGetInstanceProcAddr = nullptr;
     PFN_vkGetDeviceProcAddr fp_vkGetDeviceProcAddr = nullptr;
+    uint32_t instance_version = VKB_VK_API_VERSION_1_0;
+    uint32_t api_version = VKB_VK_API_VERSION_1_0;
```

Comment:
Now that the members are public, it'd be good to document *what* they represent, since the naming doesn't obviously indicate what is meant.

```suggestion
    // The apiVersion used to create the instance
    uint32_t instance_version = VKB_VK_API_VERSION_1_0;
    // The instance version queried from vkEnumerateInstanceVersion
    uint32_t api_version = VKB_VK_API_VERSION_1_0;
```